vnext/Microsoft.ReactNative.IntegrationTests/pch.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#define NOMINMAX
```

Comment:
Consider putting this in `React.cpp.props` which is imported by all projects

Reply:
Do we want `NOMINMAX` set for all projects?

Reply:
My preference would be yes. It's already something of opinionated defaults, e.g. WIN32_LEAN_AND_MEAN and disabling RTTI, config that needs to be shared across projects, etc.